content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -176,9 +179,16 @@ network of the consumers.
 ## Control plane models
 
 An Istio mesh uses the {{< gloss >}}control plane{{< /gloss >}} to configure all
-communication between workload instances within the mesh. You can replicate the
-control plane, and workload instances connect to any control plane instance to
-get their configuration.
+communication between workload instances within the mesh. At one extreme, you can choose
+to deploy one control plane per cluster in the mesh, while at the other extreme,
+you can host one single shared control plane for all clusters in the mesh.
+
+_Note: Each control plane will only honor configurations (such as Gateways, VirtualServices, etc.)
+that are present in its local Kubernetes cluster. If you wish to apply a configuration to all workloads in the mesh
+across control planes, it is your responsibility to replicate the configuration to the
+Kubernetes clusters associated with each control plane._
+
+### Shared control plane models
```

Comment:
I think this should move below the single cluster simple case and go right before "Multicluster deployments can also share control plane instances."

A single cluster mesh isn't really a shared control plane.

Reply:
Even if you move this header after the single cluster simple case, it still isn't appropriate because the external control plane example, later on, is also not a shared control plane model. Do we really need this heading at all?

Reply:
+1 for removing the heading